### DISCUSSION TITLE
Lato fonts only support Latin-script alphabets?

### DISCUSSION THREAD

**Discussion by rstm-sf** [QuestPDF/QuestPDF#353] [Q&A] [answered]
_Lato_ fonts only support Latin-script alphabets? 

Maybe we can replace the default fonts with other fonts? For example, _Inter_ fonts supports other

Tests:
- Lato: https://fonts.google.com/specimen/Lato/tester
- Inter: https://fonts.google.com/specimen/Inter/tester

**Comment by girlpunk** [comment]
For reference, Inter is about 4 times the size of Lato

**Reply by rstm-sf** [reply]
Okay, I'll rephrase the question.

Is there a convenient API so that you can set the default font once? Now I have to specify the font every time

**Reply by MarcinZiabek** [reply]
You can globally set the default text style: https://www.questpdf.com/api-reference/page#global-text-style

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

As @MarcinZiabek mentioned, you can set a global default font via [DefaultTextStyle](https://www.questpdf.com/api-reference/default-text-style.html). Additionally, QuestPDF now supports [font fallback](https://www.questpdf.com/api-reference/text/text-style.html), which allows automatic switching to alternative fonts for characters not available in the primary font — ideal for multilingual documents.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
